1차/윤형진_크레인_인형_뽑기.py

```diff
@@ -0,0 +1,15 @@
+def solution(board, moves):
+    stack = []
+    point = 0
+    for i in moves : # 크레인의 동선, 단 좌표로 쓰니까 1 빼야함, x좌표라 생각
+        for j in range(len(board)) : # 맨 위부터 아래로 내려옴, y좌표라 생각
+                if board[j][i - 1] != 0 :
+                    stack.append(board[j][i - 1])
+                    board[j][i - 1] = 0
```

Comment:
```suggestion
        for line in board : # 맨 위부터 아래로 내려옴, y좌표라 생각
                if line[i - 1] != 0 :
                    stack.append(line[i - 1])
                    line[i - 1] = 0
```

따로 j 값에 대해 사용하는게 없어 다음과 같이 변경하는게 더 좋을 것 같아요.

Reply:
파이썬 코드 스타일에 맞춰갈 필요가 있음  계속 해서 연습 예정